app/templates/components/active-repo-count.hbs

```diff
@@ -0,0 +1,15 @@
+<h3 class="insights-glance__title">Active Repositories</h3>
+<div class="insights-glance__stat">
+  {{#if (or isLoading activeTotalIsLoading)}}
+    {{loading-indicator}}
+  {{else}}
+    {{activeTotal}}
+  {{/if}}
+</div>
+<div class="insights-glance__chart">
+  {{#if (or isLoading (is-empty aggregateData))}}

```

Comment:
I would suggest to avoid template helpers wherever possible in favour of defining computed properties. Here's some reading about the topic: https://simplabs.com/blog/2017/03/21/on-computed-properties-vs-helpers.html

Reply:
That makes sense, change implemented 😄